boutiques_descriptors/freesurfer_mris_convert_7_4_1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "FreeSurferMrisConvert",
+  "tool-version": "v7.4.1",
+  "url": "https://surfer.nmr.mgh.harvard.edu/fswiki/mris_convert",
+  "descriptor-url": "https://github.com/aces/cbrain-plugins-neuro/blob/master/boutiques_descriptors/freesurfer_recon_all_7_4_1.json",
+  "author": "Bruce Fischl <fischl@nmr.mgh.harvard.edu>",
+  "description": "This program will convert MRI-surface data formats. These file formats are supported: ASCII: .asc; ICO: .ico, .tri; GEO: .geo; STL: .stl; VTK: .vtk.; GIFTI:  .gii.  MGH surface-encoded 'volume': .mgh, .mgz\n. Freesurfer binary format assumed for all other extensions. This is a basic version with minimal set of options.",
+  "command-line": "export FS_LICENSE=`pwd`/[LICENSE_FILE]; mris_convert [CURV_OPT] [ORIG_OPT] [PATCH_OPT] [INFILE] [OUTFILE]",
```

Comment:
Why use

```
`pwd`
```

instead of

```
$PWD
```

Reply:
I did not dig deep into distinction but my primary motive was keeping the style of the other FreeSurfer tool we have in CBRAIN, `recon_all`

Generally `pwd` seems like a more solid choice to me, because `pwd` command is independent of shell or its current state. The overhead of using this command seems to be negligible compared with image processing tool, but tell me if you would like me to change. 

Do you think I should change `pwd` to `$PWD` ?